### PR TITLE
Feedback button which redirects to github issues

### PR DESCRIPTION
**Pull request by AnishReddyRavula**
This is when artifact contains github url

image

### CODE REVIEW

File: sharing_portal/views.py [hunk 1]
```diff
@@ -632,6 +635,7 @@ def artifact(request, artifact, version_slug=None):
         "show_launch": show_launch,
         "git_content": git_content,
         "http_content": http_content,
+        "feedback_url": feedback_url,
```
Comment (super-cooper) [inline sharing_portal/views.py:638]: This will be an unresolved symbol if the artifact doesn't use git.

**Review by super-cooper — CHANGES_REQUESTED** [review]
Seems fine to me as long as we're cool with them getting a 404 on certain repos. I believe we already had that discussions.

Requesting changes because there is a critical bug, but other than that I approve.

**Comment by super-cooper** [comment]
Also, in theory, an artifact can use different git repos for different versions. Are we cool always taking the first one?

**Review by super-cooper — APPROVED** [review]

File: sharing_portal/views.py [hunk 2]
```diff
@@ -616,6 +616,10 @@ def artifact(request, artifact, version_slug=None):
             messages.error(request, message)
 
     git_content = [method for method in access_methods if method["protocol"] == "git"]
+    feedback_url = ""
+    if len(git_content) > 0:
+        # git repository is /<repository>.git format
+        feedback_url = git_content[0]["remote"].replace(".git", "/issues")
```
Comment (Mark-Powers) [inline sharing_portal/views.py:622]: This should explicitly check for github, and may need to extract the username/repo name to be correct, as the url might be `git://` too. For trovi, we use a library to do so: https://github.com/ChameleonCloud/trovi/blob/main/trovi/storage/backends/git.py#L57

**Review by Mark-Powers — CHANGES_REQUESTED** [review]
We need to be a bit cleverer about parsing the version url.

**Review by Mark-Powers — APPROVED** [review]
This looks great! I like the tests too! Did the tests require any special configuration to run? If so, it would be useful to add a short section to the readme with instructions.

**Pull request merged**